blackbox/docs/index_html.rst

```diff
@@ -31,3 +31,4 @@ Contents
    protocols/index
    enterprise/index
    administration/index
+   monitoring/index
```

Comment:
this was necessary for sphinx to build without error. every document must be referenced from within a doctree